spot_driver/src/interfaces/rclcpp_parameter_interface.cpp

```diff
@@ -196,6 +198,11 @@ bool RclcppParameterInterface::getGripperless() const {
   return declareAndGetParameter<bool>(node_, kParameterNameGripperless, kDefaultGripperless);
 }
 
+std::chrono::seconds RclcppParameterInterface::getTimeSyncTimeout() const {
+  int timeout_seconds = declareAndGetParameter<int>(node_, kParameterTimeSyncTimeout, kDefaultTimeSyncTimeout.count());
```

Comment:
```suggestion
  int timeout_seconds = declareAndGetParameter<int>(node_, kParameterTimeSyncTimeout, std::chrono::duration_cast<std::chrono::seconds>(kDefaultTimeSyncTimeout).count());
```

this gets slightly dangerous if someone decided to change `kDefaultTimeSyncTimeout` to a type like `std::chrono::milliseconds`. Without the duration cast to guarantee the type is seconds, the `count()` may return a number that's not meant for seconds (and the compiler won't catch it)